pkg/gitops/apply.go

```diff
@@ -26,7 +27,7 @@ type Applier struct {
 func (g *Applier) Run(ctx context.Context) error {
 
 	// Install Flux, Helm and Tiller. Clones the user's repo
-	err := g.FluxInstaller.Run(context.Background())
+	userInstructions, err := g.FluxInstaller.Run(context.Background())
```

Comment:
Either this or return the ssh key and form the message in the caller? I think that might be a cleaner API but I don't have a strong opinion.

Reply:
I considered this, but didn't do it as:

1. then we could potentially have the instructions diverge (unless we have a function we call for that, but that starts to be involved?)
2. we have other "commands"/functions returning status like that.